src/Components/DrillPage.js

```diff
@@ -103,34 +165,31 @@ const styles = StyleSheet.create({
     fontSize: theme.FONT_SIZE_MEDIUM,
   },
   separator: {
-    height: 10,
+    height: 15,
     borderRightWidth: 1,
     borderRightColor: theme.COLOR_PRIMARY_LIGHT,
   },
   videoLink: {
-    margin: 40,
     width: 100,
     height: 100,
     borderRadius: 50,
-    backgroundColor: theme.COLOR_PRIMARY_LIGHT,
+    backgroundColor: '#FFF933',
```

Comment:
It looks like this color is useless since we use a linear gradient that defines colors